system_baseline/openapi/api.spec.yaml

```diff
@@ -12,6 +12,18 @@ servers:
   - url: "{{ path_prefix }}{{ app_name }}/v1"
 
 paths:
+  /version:
```

Comment:
I always forget... do we have a status API?

Reply:
we do but it's not presented externally. Usually only openshift itself can access it